Makefile

```diff
@@ -61,29 +36,9 @@ clean:
 	rm -rf dist
 	mkdir dist
 	rm -rf .cache
-ifneq (,$(wildcard xiplot/bundle-dash-app.py))
-	rm xiplot/bundle-dash-app.py
```

Comment:
I guess we're no longer copying the bundling script inside the xiplot folder?

Reply:
I made the copying optional